core/src/epicli/data/common/defaults/epiphany-cluster.yml

```diff
@@ -20,6 +20,15 @@ specification:
     network:
       use_network_security_groups: True
   components:
+    repository:
+      count: 1
+      machine: repository-machine
+      configuration: default
+      subnets:
```

Comment:
Is the same subnet as K8s by purpose? From security perspective a separated subnet would not be better?

Reply:
ok